beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -86,6 +86,11 @@ type SignedValidatorRegistration struct {
 	Signature string                 `json:"signature" validate:"required,hexadecimal"`
 }
 
+type FeeRecipient struct {
+	ValidatorIndex string `json:"validator_index"`
+	FeeRecipient   string `json:"fee_recipient" hex:"true"`
```

Comment:
```suggestion
	FeeRecipient   string `json:"fee_recipient"`
```
`validate` tags missing